.gitignore

```diff
@@ -114,4 +114,5 @@ dmypy.json
 venv/ 
 env/
 .venv 
-.env
\ No newline at end of file
+.env
+migrations/
```

Comment:
Migrations are *not* usually ignored. For exaple, see this - https://www.gitignore.io/api/django

Any particular reasons?

Reply:
No. Will change it.